### ISSUE TITLE
I just updated VSCode to version 1.34 and the forceCode stopper working

### ISSUE THREAD

**Issue by sfdcdog** [celador/ForceCode#285]
I just updated VSCode to version 1.34 and the forceCode stopper working

![image](https://user-images.githubusercontent.com/29782281/58349088-36c8fd00-7e17-11e9-9822-f3c2fd0413cf.png)

I am unable to connect to Salesforce, not able to use Force commands in the command palette. I just have ForceCode extension. Should I be updating something after updating VsCode?

**Comment by daishi4u** [comment]
Please refer to issue #279 and see if it helps, then let me know

**Comment by sfdcdog** [comment]
I looked at #285 and It’s working after creating a new project.. thank you so much..

Regards,
Sahitya.

**Comment by datajock8** [comment]
I have this problem and issue #279 did not resolve by creating a new project. Still stuck after reading these different related issues and trying solutions. Still stuck. Please help.

https://github.com/forcedotcom/salesforcedx-vscode/issues/1296
https://github.com/celador/ForceCode/issues/252
https://github.com/celador/ForceCode/issues/59

**Comment by daishi4u** [comment]
@datajock8 Can you successfully login using just the sfdx cli commands? (sfdx force:auth:web:login?) If not, have you tried following the information here: https://developer.salesforce.com/docs/atlas.en-us.sfdx_setup.meta/sfdx_setup/sfdx_setup_install_cli_firewall.htm

**Comment by datajock8** [comment]
@daishi4u I am not able to login using the sfdx cli commands. I also troubleshot the firewall with my IT department with no luck after several days. They mentioned installing Netskope traffic monitoring just as my VS Code stopped working. I also updated the sfdx using update in command line. I also just updated to the new VSCode version 1.36 just to try that and still same authentication trouble.

Anything else we should try?
